apps/checkout-app/backend/payments/providers/adyen/utils.ts

```diff
@@ -0,0 +1,48 @@
+import { OrderFragment, TransactionActionEnum } from "@/graphql";
+import { Types } from "@adyen/api-library";
+
+export const getAdyenAmountFromSaleor = (float: number) => float * 100;
+export const getSaleorAmountFromAdyen = (integer: number) => integer / 100;
+
+export const mapAvailableActions = (
+  operations: Types.notification.NotificationRequestItem.OperationsEnum[]
+): TransactionActionEnum[] =>
+  operations.map((operation) => {
+    if (
```

Comment:
How bout le `switch`

Reply:
I prefer if as it's less likely to mess up and go into another case when return is omitted, TS here checks that all conditions are met